python/python/lance/vector.py

```diff
@@ -134,7 +134,9 @@ def train_ivf_centroids(
     sample_rate: int = 256,
 ) -> np.ndarray:
     """Use accelerator (GPU or MPS) to train kmeans."""
```

Comment:
nit: maybe we should rename `train_ivf_centroids_accelerator`

Reply:
also update the type signature to allow `torch.Device`

Reply:
done